src/components/DBCIDP/dbcidp.client.js

```diff
@@ -72,6 +72,32 @@ export async function getDbcidpAgencyRights(accessToken, user) {
   return result.length === 0 ? {} : [result];
 }
 
+/**
+ * Function to retrieve agency rights from DBCIDP service and return result as FORS structure to help older clients
+ *
+ * @param {string} accessToken
+ * @param {object} user information
+ * @return {array}
+ */
+export async function getDbcidpAgencyRightsAsFors(accessToken, user) {
+  const dbcidp = await getDbcidpAgencyRights(accessToken, user);
+  log.info('map DBCIDP to FORS');
+  const result = [];
+  dbcidp.forEach((agency) => {
+    if (agency.agencyId) {
+      const fors = {agencyId: agency.agencyId, rights: {}};
```

Comment:
en `const` du skriver til? Hvad er begrundelsen for at vælge const?

Reply:
const betyder vel ikke immutable, så fors variablen kan godt ændres, men ikke assignes.